stdlib/build.rs

```diff
@@ -6,10 +6,13 @@ use std::{
     path::Path,
 };
 
+mod stdlib_docs;
+
 // CONSTANTS
 // ================================================================================================
 
 const ASM_DIR_PATH: &str = "./asm";
+const STDLIB_DOC_FUNCTIONS_FILE_PATH: &str = "./docs";
```

Comment:
nit: would prefer this to be `DOC_DIR_PATH`.

Reply:
fixed